src/middlewared/middlewared/plugins/zfs.py

```diff
@@ -586,6 +588,116 @@ def query_for_quota_alert(self):
             for dataset in self.query()
         ]
 
+    def available_zvols_fast(self, options=None):
```

Comment:
If you add `@filterable` and return a list instead of a dict, you'll be able to call it like `call("zfs.pool.available_zvols_fast", [["attachment","=",None]])`

Also why `available_zvols_fast`? It seems that it can query both available and unavailable zvols ("available" in terms on being or not being used by ISCSI/VM, or is there a different meaning here?). Should we name it `query_zvols_fast`?

Return schema can be made compatible with `zfs.pool.query` to allow using it as a drop-in replacement.

Reply:
available as in being available via /sys/block. I.e. unlocked on an imported pool.

Reply:
Can `unlocked` be a better term? `query_unlocked_zvols_fast`

Reply:
Changed to "unlocked".

Reply:
"available" makes more sense to me since zvols can be unmounted